fiddler/manifest.json

```
@@ -81,7 +81,8 @@
       "auto_install": true
     },
     "dashboards": {
-      "Fiddler": "assets/dashboards/fiddler_overview.json"
+      "Fiddler v2": "assets/dashboards/fiddler_overview_v2.json",
```

```suggestion
      "Fiddler": "assets/dashboards/fiddler_overview_v2.json",
```
It's ok that you changed the actual file, but can you please keep this manifest key the same? Otherwise it treats this as a new dashboard instead of replacing the existing one 🙏

---

Sounds good. Updated as per your suggestion.